src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
+
+  checkHealth() {
+    if (this.health <= 0) {
```

Comment:
The `checkHealth` method currently only removes the animal from the `Animal.alive` array if its health is less than or equal to zero. Ensure that this method is called consistently after any operation that might change an animal's health, such as in the `Carnivore.bite` method, to maintain the integrity of the `Animal.alive` array.